lizmap/modules/lizmap/classes/qgisVectorLayer.class.php

```diff
@@ -282,6 +282,22 @@ public function getDatasourceProfile($timeout = 30, $setSearchPathFromLayer = tr
                 if (!empty($dtParams->sslmode)) {
                     $jdbParams['sslmode'] = $dtParams->sslmode;
                 }
+                // when qgis authentication config is used to authenticate a layer against, it requires to have preconfigured jdb::profile to align with the login credentials set in qgis authcfg
```

Comment:
```suggestion
                // When the QGIS authentication config is used to authenticate a layer,
                // it requires to have set up "jdb::profile" to align with the login credentials set in QGIS "authcfg"
```

The line is too long, do not use word-wrap ;-)